ercc/Makefile

```diff
@@ -14,6 +14,11 @@ decorator-plugin:
 test:
 	go test -v
 
-clean:
+clean: docker-clean
 	go clean
 
+ERCC_NAME?=ercc
+DOCKER_IMAGE?=$$(docker images | grep -- -$(ERCC_NAME)- | awk '{print $$1;}')
+
+docker-clean: docker-stop
```

Comment:
forgot the docker-stop target?

Reply:
yes and no. i did have that target but then marcus convinced me that for ercc it doesn't make sense to have docker-start and docker-stop and so removed it, but forgot to remove also this dependency ...

Reply:
ok; will take care in #52